debug.log

```diff
@@ -0,0 +1 @@
+[0114/095535.306:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
Can you remove this? Looks accidentally added

Reply:
removed